Rakefile

```diff
@@ -96,6 +96,30 @@ namespace :travis do
 end
 
 namespace :docs do
+  desc "Generate markdown docs for Bolt's transport configuration options"
+  task :config_reference do
+    @transports = { options: {}, defaults: {} }
+    @global = {}
```

Comment:
I think `@transports` is the only var that gets calculated here. Everything else can be set in it's definition for readability and brevity.
```
@global = { options: Bolt::Config::OPTIONS,
defaults: Bolt::Config::DEFAULT_OPTIONS
}
...
```